Specify `nofail` when adding entries to fstab

paulmey opened the pull request:
Any error trying to mount a filesystem marked as auto, may hang the VM during boot, preventing the customer from self-mitigating. Adding the nofail option ignores the error during boot, allowing the customer to fix the faulty entry or retry the fs mount.

azurecla commented:
Hi __@GaugeField__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abhic). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

paulmey committed:
Specify `nofail` when adding entries to fstab

Any error trying to mount a filesystem marked as auto, may hang the VM during boot, preventing the customer from self-mitigating. Adding the nofail option ignores the error during boot, allowing the customer to fix the faulty entry or retry the fs mount.

Lisaco88 committed:
Removed repeated word
